lib/quiz/multiple_choice.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+
+const Map<String, dynamic> _homework = {
+  'data': [
```

Comment:
You dont need a data->array map here. Directly have your map:
    {
      'image': 'lion',
      'questions': "#This animal is a carnivorous reptile.",
      'answer': 'lion',
      'choices': ["Cat", "Sheep", "lion", "Cow"],
    },